hydradx/model/amm/omnipool_amm.py

```diff
@@ -1,6 +1,66 @@
 import copy
 import string
 
+import pytest
```

Comment:
I don't think we should be doing anything with pytest in omnipool_amm.py, this should be just in test files